tests/e2e/authoring-cart.spec.js

```
@@ -0,0 +1,443 @@
+    // edited things and staged none of them with no way in at all.
+    await expect(page.getByTestId('authoring-cart')).toBeVisible()
+    await page.getByTestId('authoring-cart-toggle').click()
+    await expect(page.getByTestId('authoring-cart')).toHaveCount(0)
+    await page.getByTestId('authoring-cart-toggle').click()
+    await expect(page.getByTestId('authoring-cart')).toBeVisible()
+  })
+
+  test('a deletion reads as a deletion, not as an edit', async ({ page }) => {
+    await page.goto('/', { waitUntil: 'networkidle' })
+    await page.evaluate(() =>
+      window.$nuxt.$store.dispatch('authoringCart/stageDeletion', {
```

ESLint

[error] 'window' is not defined.

(no-undef)